lib/fuzzy_tools/helpers.rb

```diff
@@ -30,8 +30,6 @@ def ngrams(str, n)
       (0..str.length - n).map { |i| str[i,n] }
     end
 
-    if RUBY_DESCRIPTION !~ /^ruby/ # rbx, jruby
-
       SOUNDEX_LETTERS_TO_CODES = {
```

Comment:
Please align the lines that used to be inside `if` with the rest of the code (basically, unindent 1 level)